contract/scripts/get-token-uri.ts

```diff
@@ -0,0 +1,17 @@
+import { BigNumberish, Contract, ethers } from "ethers";
+import * as apeTokenJson from "../artifacts/contracts/ApeToken.sol/ApeToken.json"
+import { ApeToken } from "../typechain";
+async function main(signer: ethers.Wallet, apeTokenAddress: string, tokenId: BigNumberish) {
+  // =================================================================
+  // Mint NF
```

Comment:
@allen-hsu 
This should be `Getting Token URI` , not `Mint NF`  😄

Reply:
oh , copy paste and forgot to modify